src/test/java/com/puppycrawl/tools/checkstyle/checks/whitespace/SeparatorWrapCheckTest.java

```diff
@@ -58,19 +58,19 @@ public void testDot()
         checkConfig.addAttribute("option", "NL");
         checkConfig.addAttribute("tokens", "DOT");
         final String[] expected = {
-            "31:10: " + getCheckMessage(MSG_LINE_NEW, "."),
+            "29:10: " + getCheckMessage(MSG_LINE_NEW, "."),
```

Comment:
@rnveach 
Agree. Lest postpone my idea till we resolve all issues related to splitting of input files.